src/client/kotlin/tech/thatgravyboat/skyblockapi/api/events/base/Subscription.kt

```diff
@@ -5,6 +5,11 @@ import kotlin.reflect.KClass
 @Retention(AnnotationRetention.RUNTIME)
 @Target(AnnotationTarget.FUNCTION)
 annotation class Subscription(
+    /**
+     * The event that will be received, only is required if there are no parameters.
+     */
+    vararg val event: KClass<out SkyBlockEvent> = [],
```

Comment:
I still need to verify if that still actually break because the bytecode for annotations is very weird